docs/Data-Loading.md

```diff
@@ -45,6 +45,7 @@ There are just a few rules to follow:
 - cancer type files can be mandatory if the study is referring to a cancer type that does not yet exist in the DB.
 - meta files can be named anything, as long as it starts or ends with name 'meta'. E.g. meta_test, meta.test, test.meta are all fine; metal_test and metastudy are wrong.
 - data files can be named anything and are referenced by a property `data_filename` set in the meta file. 
+- gene panel filenames should start with meta_gene_panel. E.g. meta_gene_panel_impact410.
```

Comment:
If I remember , I had to have the file start with meta_ so that it would be processed by the validator.  I think I added _gene_panel for consistency, similar to how we have adopted meta_CNA, meta_log2CNA, etc. Technically, there is no reason for the added restriction.